package.json

```diff
@@ -1,14 +1,11 @@
 {
-	"name": "@divriots/starter-systemui",
+	"name": "@divriots/starter-react-sui",
 	"version": "0.0.0",
 	"description": "",
 	"license": "ISC",
 	"type": "module",
 	"homepage": "{{ homepage }}",
 	"scripts": {
-		"serve": "dsd-cli serve z99ptYZyvMOGDivUlP9u --open"
-	},
-	"devDependencies": {
-		"dsd-cli": "latest"
+		"serve": "npx backlight serve z99ptYZyvMOGDivUlP9u --open"
 	}
-}
+}
```

Comment:
didn't we want to include all dependencies explicitly for perf reasons and because it's less magic? I'm not suggesting to include `backlight`, it's great to use it via `npx`, I'm talking mostly about what you import in various files

Reply:
it's been brought up in Discord, so let's align the approach there with other people